src/WatchDogs.Contracts/IRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Contracts;
```

Comment:
Please do put empty lines between namespaces and classes

Reply:
That f***ing references tag creates an illusion that there is one already. Fixed.